app/scripts/modules/google/src/serverGroup/configure/wizard/loadBalancingPolicy/loadBalancingPolicySelector.component.ts

```diff
@@ -41,19 +41,20 @@ class GceLoadBalancingPolicySelectorController implements ng.IComponentControlle
     /*
     * Three cases:
     *   - If we have only HTTP(S) load balancers, our balancing mode can be RATE or UTILIZATION.
-    *   - If we have only SSL load balancers, our balancing mode can be CONNECTION or UTILIZATION.
+    *   - If we have only SSL/TCP load balancers, our balancing mode can be CONNECTION or UTILIZATION.
     *   - If we have both, only UTILIZATION.
     * */
     if (has(this, 'command.backingData.filtered.loadBalancerIndex')) {
       const index = this.command.backingData.filtered.loadBalancerIndex;
       const selected = this.command.loadBalancers;
 
       const hasSsl = selected.find((loadBalancer: any) => get(index[loadBalancer], 'loadBalancerType') === 'SSL');
+      const hasTcp = selected.find((loadBalancer: any) => get(index[loadBalancer], 'loadBalancerType') === 'TCP');
       const hasHttp = selected.find((loadBalancer: any) => get(index[loadBalancer], 'loadBalancerType') === 'HTTP');
 
-      if ((hasSsl && hasHttp)) {
+      if (((hasSsl || hasTcp) && hasHttp)) {
```

Comment:
Please remove the extra set of parentheses here (the fault is in the original code, but no reason to duplicate it).